apiserver/pkg/swagger/datafile.go

```diff
@@ -5,7 +5,8 @@
 // third_party/swagger-ui/index.css (202B)
 // third_party/swagger-ui/index.html (734B)
 // third_party/swagger-ui/oauth2-redirect.html (2.715kB)
-// third_party/swagger-ui/swagger-initializer.js (1.013kB)
+// third_party/swagger-ui/swagger-initializer.js (539B)
```

Comment:
Just for my own knowledge (sorry for my ignorance), are these changes (datafile.go and swagger-initializer.js) auto-generated?